tests/test_playbooks/tasks/location.yml

```diff
@@ -21,6 +21,12 @@
   when: expected_change is defined
 - assert:
     fail_msg: "Ensuring location is {{ location_state }} didn't fail! (expected_error: {{ expected_error | default('unknown') }})"
+    that:
+      - result is succeeded
+  when:
+    - expected_error is not defined or not expected_error
```

Comment:
Can we use `expected_error is defined` as a condition for ignore_errors?

Reply:
maybe? I'll try

Reply:
YES!